web/thesauruses/javascript/control_structures.json

```diff
@@ -10,88 +10,56 @@
       "if_else_conditional",
       "if_elseif_conditional",
       "if_elseif_else_conditional",
+      "switch_statement",
       "ternary_conditional"
     ],
     "Loops": [
       "while_loop",
       "do_while_loop",
-      "until_loop",
-      "do_until_loop",
       "for_loop",
       "foreach_loop"
     ],
     "Iterations": [
-      "each_iteration",
       "map_iteration",
-      "filter_iteration",
-      "fold_iteration",
-      "recursion_iteration"
+      "filter_iteration"
     ]
   },
   "control_structures": {
     "if_conditional": {
-      "name": "If conditional",
-      "code": "if(condition){code block to be executed}"
+      "code": "if (condition) {\n    code block to be executed\n}"
     },
     "if_else_conditional": {
-      "name": "If/Else conditional",
-      "code": "if(condition){code block to be executed} else {code block to be executed}"
+      "code": "if (condition) {\n    code block to be executed\n} else {\n    code block to be executed\n}"
     },
     "if_elseif_conditional": {
-      "name": "If/ElseIf conditional",
-      "code": "if(condition){code block to be executed} else if(condition){code block to be executed}"
+      "code": "if (condition) {\n    code block to be executed\n} else if (condition) {\n    code block to be executed\n}"
     },
     "if_elseif_else_conditional": {
-      "name": "If/ElseIf/Else conditional",
-      "code": "if(condition){code block to be executed} else if(condition){code block to be executed} else {code block to be executed}"
+      "code": "if (condition) {\n    code block to be executed\n} else if (condition) {\n    code block to be executed\n} else {\n    code block to be executed\n}"
```

Comment:
"Code block to be executed" should probably something like "statement()" or something actually compilable. If you want to add a comment you can but I don't know that this needs it.